src/probeinterface/neuropixels_tools.py

```diff
@@ -515,10 +57,208 @@
     "NP1121": "1121",  # Ultra probe - beta configuration
     # Opto
     "NP1300": "1300",  # Opto probe
-    # missing NP1200
-    # missing NXT3000
 }
 
+probe_type_to_probe_part_number = {v: k for k, v in probe_part_number_to_probe_type.items()}
+
+imro_field_to_pi_field = {
+    "ap_gain": "ap_gains",
+    "ap_hipas_flt": "ap_hp_filters",
+    "bank": "banks",
+    "bank_mask": "banks",
+    "channel": "channel_ids",
+    "electrode": "elec_ids",
+    "lf_gain": "lf_gains",
+    "ref_id": "references",
+    "shank": "shank_id",
+    "group": "group",
+    "bankA": "bankA",
+    "bankB": "bankB",
+}
+
+pi_to_pt_names = {
+    "x_pitch": "electrode_pitch_horz_um",
+    "y_pitch": "electrode_pitch_vert_um",
+    "contact_width": "electrode_size_horz_direction_um",
+    "shank_pitch": "shank_pitch_um",
+    "shank_number": "num_shanks",
+    "ncols_per_shank": "cols_per_shank",
+    "nrows_per_shank": "rows_per_shank",
+    "adc_bit_depth": "adc_bit_depth",
+    "model_name": "description",
+    "num_readout_channels": "num_readout_channels",
+    "shank_width_um": "shank_width_um",
+    "tip_length_um": "tip_length_um",
+}
+
+
+def make_npx_description(probe_part_number):
+    """
+    Extracts probe metadata from the `probeinterface/resources/probe_features.json` file and converts
+    to probeinterface syntax. File is maintained by Bill Karsh in ProbeTable
+    (https://github.com/billkarsh/ProbeTable/tree/main).
+
+    Parameters
+    ----------
+    probe_part_number : str
+        The part number of the probe e.g. 'NP2013'.
+
+    Returns
+    -------
+    pi_metadata : dict
+        Dictionary containing metadata about NeuroPixels probes using ProbeInterface syntax.
+    """
+
+    is_phase3a = False
+    # These are all prototype NP1.0 probes, not contained in ProbeTable
+    if probe_part_number in ["PRB_1_4_0480_1", "PRB_1_4_0480_1_C", "PRB_1_2_0480_2", None]:
```

Comment:
`PRB_1_4_0480_1` and `PRB_1_4_0480_1_C` are on the table, why are we fetching `NP1010` instead for those?